lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -384,6 +392,24 @@ describe('BasicLtiLaunchApp', () => {
             groupsCallReject = reject;
           })
         );
+    };
+
+    beforeEach(() => {
+      // Will attempt to fetch the 1. content url and 2. groups.
```

Comment:
It's not immediately clear what "Will attempt to fetch" refers to here. I presume it should say something like "When `BasicLtiLaunchApp` is rendered, it will attempt..."?